### PR TITLE
code clean up

### PR DESCRIPTION
**Pull request by jcurlee**

### CODE REVIEW

File: triangle_facts/triangle_facts.rb [hunk 1]
```diff
@@ -1,58 +1,70 @@
...
+  def equilateral?
+    side1 == side2 && side2 == side3
+  end
+
+  def isosceles?
+    [side1, side2, side3].uniq.length == 2
+  end
+
+  def scalene?
+    if !(equilateral? || isosceles?)
```
Comment (oharace) [inline triangle_facts/triangle_facts.rb]: What if, instead we simplified it to `!(equilateral? || isosceles?)`? That would make it even better!

File: triangle_facts/triangle_facts.rb [hunk 2]
```diff
@@ -1,58 +1,70 @@
...
+        'This triangle is isosceles! Also, that word is hard to type.'
+      when scalene?
+        'This triangle is scalene and mathematically boring.'
+    end
+  end
+
+  def recite_angle_facts
+    angles = calculate_angles(side1, side2, side3)
 
+    result = "\nThe angles of this triangle are " + angles.join(',')
```
Comment (oharace) [inline triangle_facts/triangle_facts.rb]: Using `puts` statements seems better here in my opinion. First, you can just call `puts` at a few places where you need to print a string out. Second, `puts` takes care of newline characters, so you don't need to worry about including them in your strings.

File: triangle_facts/triangle_facts.rb [hunk 3]
```diff
@@ -1,58 +1,70 @@
...
 
-triangles = [
-	[5,5,5],
-	[5,12,13],
-]
-triangles.each { |sides|
-	tri = Triangle.new(*sides)
-	tri.recite_facts
-}
+    if angles.include? 90
```
Comment (oharace) [inline triangle_facts/triangle_facts.rb]: In most circumstances, you should put parameters in parentheses. There are exceptions (like `puts`), but most of the time you should include the parentheses.

File: triangle_facts/triangle_facts.rb [hunk 4]
```diff
@@ -1,58 +1,70 @@
...
+  def radians_to_degrees(rads)
+    (rads * 180 / Math::PI).round
+  end
+end
+
+[
+  [5, 5, 5],
+  [5, 12, 13],
+].each do |sides|
+  tri = Triangle.new(*sides)
```
Comment (oharace) [inline triangle_facts/triangle_facts.rb]: That's neat, I never though of using the splat character here to split the sides into separate arguments. Cool.

File: triangle_facts/triangle_facts.rb [hunk 5]
```diff
@@ -1,58 +1,70 @@
...
+
+  def scalene?
+    if !(equilateral? || isosceles?)
+      true
+    else
+      false
+    end
+  end
+
+  def recite_facts
```
Comment (oharace) [inline triangle_facts/triangle_facts.rb]: In the method below this one, you're specific, but here is pretty generic. Maybe a better name would be `recite_triangle_type`.

**Comment by oharace** [comment]
Nice work! You did a great job simplifying this code.

**Comment by paulghaddad** [comment]
@jcurlee Nice work! You're good to move on.

**Pull request closed without merging**